pallets/msa/Cargo.toml

```diff
@@ -26,6 +26,7 @@ sp-std = {default-features = false, git = "https://github.com/paritytech/substra
 
 # Frequency related dependencies
 common-primitives = {default-features = false, path = "../../common/primitives"}
+common-runtime = {path = '../../runtime/common', default-features = false}
```

Comment:
I think this is also extra.

Reply:
Removed which also removed from Cargo.lock file.